src/components/App.tsx

```diff
@@ -561,6 +568,10 @@ class App extends React.Component<ExcalidrawProps, AppState> {
           name = this.props.name;
         }
 
+        if (typeof this.props.canvasActions !== "undefined") {
+          canvasActions = { ...canvasActions, ...this.props.canvasActions };
```

Comment:
is this needed ? in this case its just show or hide so this doesn't have any value in `initialData`